### ISSUE TITLE
Rubocop should warn about wrong indenting

### ISSUE THREAD

**Issue by markijbema** [rubocop/rubocop#428]
Rubocop did not give me a warning on indenting a method with four spaces:

``` ruby
        def allowed_name(method_name)
            method_name.match(/^[A-Z]/)
        end
```

I think it should
